README.md

```diff
@@ -57,158 +47,145 @@ Then commit `package.json` & put `/node_modules` into `.gitignore`. Next time
 
-This will install utilities and create shourtcut `lv_i18n` to call from terminal.
-You can also use local install and type explicit path to `lv_i18n.js`:
+Example:
 
-```sh
-npm install lv_i18n
+```yml
+'en-GB':
+  title1: Main menu
+  user_logged_in:
+    one: One user is logged in
+    other: %d users are logged in
 ```
+## Run compile to convert the yml files to a C and H file
 
-All "actions" use similar pattern:
-
-- Load source C/CPP files and extract phrases.
-- Load yaml translations to understand existing progress.
-- Do something useful (command):
-  - Fill yaml files with new phrases to translate
-  - Check errors and inconsistencies
-  - Generate ("compile") `.c` file with optimized translation data.
+Once you have the translations in the `yml` files you only need to run the `compile` to generate a C and H files from the `yml` files. No other library will be required to get the translation with `_()` and `_p`.
 
-See help:
+Running `compile`:
 
 ```sh
-lv_i18n -h
-lv_i18n extract -h
-lv_i18n compile -h
+lv_i18n compile -t 'translations/*.yml' -o 'src/lv_i18n'
 ```
 
-Examples:
+The deafult local is `en-GB` but you change it with `-l 'language-code'`.
 
+## Follow modifications in the source code
+To change a text id in the `yml` files use:
 ```sh
-lv_i18n extract -s 'src/**/*.+(c|cpp|h|hpp)' -s inc/**/* -t src/i18n/*.yml
-lv_i18n compile -t src/i18n/*.yml -o src/translations.c
 lv_i18n rename -t src/i18n/*.yml --from 'Hillo wold' --to 'Hello world!'
 ```
 
-[Glob expressions](https://github.com/isaacs/node-glob#glob-primer) supported.
+## C API
 
-Note for Windows: use *only forward-slashes* in glob expressions.
+#### int lv_i18n_init(const lv_i18n_lang_pack_t * langs) 
+Attach generated translations to be used by `lv_i18n_get_text()`. 
 
+- _return_ - 0 on success, -1 on fail. 
 
-Translation files format
-------------------------
+___
 
-Translations are stored in `.yaml` (or `.json`) files. Structure is similar to
-one in [Ruby on Rails](https://guides.rubyonrails.org/i18n.html), with minor
-simplifications:
+#### int lv_i18n_set_locale(const char * l_name)
+Set locale to be used by `lv_i18n_get_text()`.  
 
-```yaml
-# Default locale
-'en-GB':
-  # May be useful if shortcuts used instead of real texts.
-  # For example: `msg_file_missed`. When real texts used, default locale
-  # doesn't need translation for singular phrases (leave "null" here)
-  Settings: ~
-  # Plurals - should be filled
-  Nut:
-    # Valid plural form names are `zero`, `one`, `two`, `few`, `many`, `other`
-    # Each language has specific set of required forms.
-    one: Nut
-    many: Nuts
-  'LVGL is awesome!': ~
-
-# Example of additional locale. Usually, placed to separate file.
-'ru-RU':
-  Settings: Настройки
-  Nut:
-    one: Гайка
-    few: Гайки
-    many: Гаек
-  'LVGL is awesome!': ~ # null (still untranslated), added by extractor
-```
+- _l_name_ - locale name (`en-GB`, `ru-RU`). You can use language codes only (`en`, `ru`) instead.
```

Comment:
`You can use language codes only (`en`, `ru`) instead.` - seems confusing here (as "can search full name by short one).

Reply:
It was written by you and I didn't change it. So you can use `en` if you have `en` in the yml file too?

Reply:
I mean, phrase is correct, but placed in "wrong" place - give strange result IMO.

Yes, you can use `en`, the only difference - have to define it in compiler option, because it's default locale is `en-GB`.

Reply:
I see. Updated: https://github.com/littlevgl/lv_i18n/commit/cc848a15c48a7aed7c379a8c70dd310c08dcf8b6